client/src/res/values/joyride.js

```diff
@@ -30,13 +30,15 @@ export const JOYRIDE_STEPS = [
           spotlightClicks: true,
```

Comment:
Clicking next on the pie chart step without clicking on the pie chart leaves it as a beacon on the Lock California step (not sure why it jumps back to there, maybe its the closest non-beacon-disabled)

Reply:
I can't get this to repeat, for me it stays on the pie chart step when you click next